pkg/mkverification-raw-efi/verify

```diff
@@ -373,9 +373,11 @@ do
    fi
 done
 
+# shellcheck disable=SC2016
+: ' # Temporarily commented out
```

Comment:
What is this? Doesn't seem readable to me.  I suggested if false which is readable but I'm open to other approaches which result in readable and understandable code. But not obfuscation like this.